sgr.go

```diff
@@ -89,6 +89,10 @@ func handleSgr(parser *ansi.Parser) (string, error) { //nolint:unparam
 			str += fmt.Sprintf("Background color: %d", readColor(&i, parser.Params))
 		case 49:
 			str += "Default background color"
+		case 53:
+			str += "Overline"
+		case 55:
+			str += "No overline"
```

Comment:
These are not widely supported :/

Reply:
It's a sad thing. On the bright side, iTerm and VTE (and Konsole, and a couple others) do. Very handy for status lines.

Reply:
Even if they're not widely supported I can see them being useful, both from a completeness standpoint and for the sake of simply understanding what they are.

Reply:
@rbanffy It would be really helpful to implement these SGR arguments in `x/ansi`